### PR TITLE
[Merged by Bors] - doc: Mark named theorems including Compactness Theorem, Halting Problem, Tarski Vaught, Łoś

### PR DESCRIPTION
**Pull request by hmonroe**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]
bors d+

Don't forget to fix the long line.

File: Mathlib/Computability/Halting.lean [hunk 1]
```diff
@@ -230,10 +230,12 @@ theorem rice₂ (C : Set Code) (H : ∀ cf cg, eval cf = eval cg → (cf ∈ C 
             exact ⟨by infer_instance, Computable.const _⟩ }⟩
 #align computable_pred.rice₂ ComputablePred.rice₂
 
+/-- The Halting problem is recursively enumerable */
```
Comment (eric-wieser) [inline Mathlib/Computability/Halting.lean:233]: This is a syntax error

Comment (hmonroe) [reply Mathlib/Computability/Halting.lean]: Apologies I should not have applied the Awaiting Review label until checks were complete—I am looking at the checks. I will be making commits from VS Code going forward

**Pull request closed without merging**